Computers/Computers/Program.cs

```diff
@@ -0,0 +1,278 @@
+                                departments[i][j].CpuFrequency = 3;
+                                departments[i][j].Memory = 16;
+                                departments[i][j].Hdd = 2048;
+                            }
+                            break;
+                        default:
+                            break;
+                    }
+                }
+            }
+            // 6) count total number of every type of computers
+            // 7) count total number of all computers
+            // Note: use loops and if-else statements
+            // Note: use the same loop for 6) and 7)
+            var desktopQuantity = 0;
+            var laptopQuantity = 0;
+            var serverQuantity = 0;
+            var computerQuantity = 0;
+            for (int i = 0; i < departments.Length; i++)
+            {
+                for (int j = 0; j < departments[i].Length; j++)
+                {
+                    computerQuantity += 1;
+                    switch (departments[i][j].ComputerType)
+                    {
+                        case ComputerTypeEnum.Desktop:
+                            desktopQuantity += 1;
+                            break;
+                        case ComputerTypeEnum.Laptop:
+                            laptopQuantity += 1;
+                            break;
+                        case ComputerTypeEnum.Server:
+                            serverQuantity += 1;
+                            break;
+                    }
+                }
+
+            }
+
+            Console.WriteLine($"The total number of {ComputerTypeEnum.Desktop}s in departments is: {desktopQuantity}");
+            Console.WriteLine($"The total number of {ComputerTypeEnum.Laptop}s in departments is: {laptopQuantity}");
+            Console.WriteLine($"The total number of {ComputerTypeEnum.Server}s in departments is: {serverQuantity}");
+            Console.WriteLine($"The total number of all computers in departments is: {computerQuantity}");
+            Console.WriteLine();
+
+            // 8) find computer with the largest storage (HDD) - 
+            // compare HHD of every computer between each other; 
+            // find position of this computer in array (indexes)
+            // Note: use loops and if-else statements
+            var index1 = 0;
+            var index2 = 0;
+            var largestStorage = 0;
+            for (int i = 0; i < departments.Length; i++)
+            {
+                for (int j = 0; j < departments[i].Length; j++)
+                {
+                    if (departments[i][j].Hdd > largestStorage)
+                    {
+                        largestStorage = departments[i][j].Hdd;
+                        index1 = i;
+                        index2 = j;
+                    }
+                }
+            }
+
+            Console.WriteLine($"Computer with the largest storage is located in the array at positions {index1}, {index2}");
```

Comment:
User can't understand what is in array at position 0,0

Reply:
I completly agree with you. It doesn`t make sense for me either. I just somehow tried to implement that is written in lines 164-167

Reply:
Ok, just question of philosophy